addons/mission/functions/fnc_monitorUnits.sqf

```diff
@@ -21,9 +21,15 @@ params [["_type", 0]];
 
 switch (_type) do {
     case 0: {
-        format ["West: %1|East: %2|Indep: %3|Civ: %4|Player: %5", west countSide allUnits, east countSide allUnits, resistance countside allUnits, civilian countSide allUnits, count playableUnits]
+        private _west = west countSide allUnits - playableUnits;
+        private _east = east countSide allUnits - playableUnits;
+        private _resistance = resistance countSide allUnits - playableUnits;
+        private _civilian = civilian countside allUnits - playableUnits;
+        format ["West: %1|East: %2|Indep: %3|Civ: %4|Player: %5", _west, _east, _resistance , _civilian, count playableUnits]
     };
     case 1: {
-        format ["Active: %1|Inactive: %2|Agents: %3|FPS: %4", count (allUnits select {simulationEnabled _x}), count (allUnits select {!simulationEnabled _x}), count agents, diag_fps]
+        private _active = count (allUnits - playableUnits select {simulationEnabled _x});
```

Comment:
```suggestion
        private _active = count ((allUnits - playableUnits) select {simulationEnabled _x});
```

Reply:
Same as above, this might be faster: `count (allUnits select {simulationEnabled _x}) - count playableUnits` (as  `playableUnits are almost always simulation enabled).

Reply:
this is much faster
`private _active = (count allUnits) - count (playableUnits select {simulationEnabled _x});`